pubsub/gossipsub/README.md

```diff
@@ -46,17 +49,17 @@ profiles.
 
 The initial pubsub experiment in libp2p was `floodsub`.
 It implements pubsub in the most basic manner, with two defining aspects:
-- ambient peer discovery.
-- most basic routing; flooding.
+- ambient peer discovery; and
```

Comment:
i don't think it's correct to have this trailing and here -- it's a bullet list already, so it's implied.

Reply:
It is correct. See e.g. http://www.grammar-monster.com/lessons/using_bullet_points.htm.